docs/model_zoo.md

```diff
@@ -207,6 +211,17 @@ We also provide the [checkpoint](http://download.openmmlab.com/mmdetection/v2.0/
 | [Detectron](https://github.com/facebookresearch/Detectron) | 19 |
 | [matterport/Mask_RCNN](https://github.com/matterport/Mask_RCNN/) | 14 |
 
+### Inference Speed Benchmark
+
+We provide [benchmark.py](https://github.com/open-mmlab/mmdetection/blob/master/tools/benchmark.py) to benchmark the latency.
+The model would be benchmarked with 2000 images and take the average, you can change the output log interval (defaults: 50) by setting `LOG-INTERVAL`.
```

Comment:
The script benchmarks the model with 2000 images and calculates the average time ignoring first xxx images.